### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/deploy-app/_index.md

### PR DESCRIPTION
**Pull request by rajatgupta24**
Hey folks, this is a PR for hi localization of the file: `content/hi/docs/tutorials/kubernetes-basics/deploy-app/_index.md`
PR is linked under this issue #29353

/language hi

### CODE REVIEW

**Comment by mittalyashu** [comment]
@rajatgupta24 Is seems like the file is not translated?

**Comment by rajatgupta24** [comment]
> @rajatgupta24 Is seems like the file is not translated?

Looks like I push the wrong branch, I'll change it, thanks for reviewing

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/_index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: ऐप डिप्लॉय करें
```
Comment (mittalyashu) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/_index.md:2]: Can you use "Application deployment" or "Deploy application" to localise in hindi?

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tutorials/kubernetes-basics/deploy-app/_index.md

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @rajatgupta24 :)

**Pull request merged**